src/config.js

```diff
@@ -0,0 +1,14 @@
+export const BASE_URL = 'http://10.58.52.209:3000';
+
+export const APIS = {
+  redirect: `${BASE_URL}/users/kakao/login`,
+};
+
+export const KAKAO_BASE_URL = 'https://kauth.kakao.com';
+
+export const REDIRECT_URI = 'http://localhost:3000/auth/kakao/callback';
+
+export const KAKAO_APIS = {
+  auth: `${KAKAO_BASE_URL}/oauth/authorize?client_`,
+  getToken: `${KAKAO_BASE_URL}/oauth/token?grant_type=authorization_code&client_`,
+};
```

Comment:
```suggestion
export const BASE_URL = 'http://10.58.52.209:3000';
export const KAKAO_BASE_URL = 'https://kauth.kakao.com';
export const REDIRECT_URI = 'http://localhost:3000/auth/kakao/callback';

export const API = {
  auth: `${KAKAO_BASE_URL}/oauth/authorize?client_`,
  getToken: `${KAKAO_BASE_URL}/oauth/token?grant_type=authorization_code&client_`,
  redirect: `${BASE_URL}/users/kakao/login`,
};
```
이렇게 정리해서 endpoint별 묶음을 만들 수 있겠네요